README.md

```diff
@@ -9,10 +9,12 @@
 3. `git clone https://github.com/effectindex/EffectIndex`
 4. `cd EffectIndex`
 5. `npm install`
-6. Create a `.env` file in the `EffectIndex` directory with the following contents:
+6. Create a `.env` file in the `EffectIndex` directory with the following contents (please reference .env.example):
```

Comment:
I'd rather an instruction here telling the user to `cp .env.example .env` and to edit it.